### PR TITLE
Improved Feat/wsse auth

### PR DESCRIPTION
**Pull request by sanjai0py**
# Description

- Resolved conflicts
- Improved the WSSE auth headers creation function to use the `node:crypto` package instead of the `crypto-js` package as support for it had been [stopped](https://www.npmjs.com/package/crypto-js#discontinued)

### Contribution Checklist:

- [x] **The pull request only addresses one issue or adds one feature.**
- [x] **The pull request does not introduce any breaking changes**
- [ ] **I have added screenshots or gifs to help explain the change if applicable.**
- [x] **I have read the [contribution guidelines](https://github.com/usebruno/bruno/blob/main/contributing.md).**
- [x] **Create an issue and link to the pull request.**

Note: Keeping the PR small and focused helps make it easier to review and merge. If you have multiple changes you want to make, please consider submitting them as separate pull requests.

### Publishing to New Package Managers

Please see [here](../publishing.md) for more information.

### CODE REVIEW

**Comment by helloanoop** [comment]
Merged! Thanks @sanjai0py !

File: packages/bruno-electron/src/ipc/network/prepare-request.js [hunk 1]
```diff
@@ -61,18 +221,30 @@ const setAuthHeaders = (axiosRequest, request, collectionRoot) => {
...
-          'auth.wsse.user'
-        )}", PasswordDigest="${digest}", Created="${ts}", nonce="${nonce}"`;
+        const user = get(request, 'auth.wsse.user', '');
+        const password = get(request, 'auth.wsse.password', '');
+
+        const ts = new Date().toISOString();
+        const nonce = crypto.randomBytes(16).toString('base64');
+
+        // Create the password digest using SHA-256
+        const hash = crypto.createHash('sha256');
```
Comment (dwolter-emarsys) [inline packages/bruno-electron/src/ipc/network/prepare-request.js:231]: This logic is no longer correct- the password digest must be computed with a SHA1 hash to be a valid WSSE header

Comment (sanjai0py) [reply packages/bruno-electron/src/ipc/network/prepare-request.js:231]: Hey, I made the changes because the SHA-1 algorithm was more prone to attacks.

Comment (dwolter-emarsys) [reply packages/bruno-electron/src/ipc/network/prepare-request.js:231]: True as that may be, API servers don't tend to appreciate arbitrary changes to the standards they're expecting :)

I will open a new feature branch and new pull request to swap to SHA1. If it's not SHA1 it's not WSSE

**Pull request merged**